Nos/Views/PreviewData.swift

```diff
@@ -55,7 +55,7 @@ struct PreviewData {
     }()
     
     lazy var bob: Author = {
-        let author = Author(context: previewContext)
+        let author = try! Author.findOrCreate(by: KeyFixture.bob.publicKeyHex, context: previewContext)
```

Comment:
I think this was a bug that `bob` wasn't in Core Data. `alice` and `eve` were though, so I copied what they did.